tests/Doctrine/Tests/ORM/Tools/Console/Command/InfoCommandTest.php

```diff
@@ -41,15 +45,112 @@ protected function setUp()
 
         $this->application->add($command);
 
-        $this->command    = $this->application->find('orm:info');
-        $this->tester     = new CommandTester($command);
+        $this->command = $this->application->find('orm:info');
+        $this->tester  = new CommandTester($command);
     }
 
     public function testListAllClasses()
     {
         $this->tester->execute(['command' => $this->command->getName()]);
 
-        $this->assertContains(AttractionInfo::class, $this->tester->getDisplay());
-        $this->assertContains(City::class, $this->tester->getDisplay());
+        self::assertContains(AttractionInfo::class, $this->tester->getDisplay());
+        self::assertContains(City::class, $this->tester->getDisplay());
+    }
+
+    public function testEmptyEntityClassNames()
```

Comment:
I tried to overwrite the `HelperSet`with my mocked `$em` like this, but it doesn't work:
```php
// overwrite HelperSet to mock getAllClassNames()
         $this->application->setHelperSet(
             new HelperSet(['em' => new EntityManagerHelper($em)])
         );
```

because of this I need to redo the whole `setUp` logic in my tests

Reply:
okay, let's give it a shot but if it gets too complicated it means that we have a design issue and wouldn't be really correct to fix it in the PR (trade-offs)

Reply:
I am done with the test, but will force push once you finished your review.

Reply:
You can push it, I didn't start the review yet

Reply:
done